### ISSUE TITLE
Addition of decaying learning rate

### ISSUE THREAD

**Issue by JanFidor** [unit8co/darts#1680]
Hi! After diving a little deeper into CL and checking out some prediction kaggle notebooks, I noticed that in both cases decaying learning rate seemed to be yielding decent improvements under certain conditions. I think it would be a decent improvement for darts, but I'm not certain if implementation would be straight forward, let me know what you think!

**Comment by dennisbader** [comment]
Not entirely sure what you mean by CL, but if you are talking about learning rate scheduling for TorchForecastingModels, we do support this with `lr_scheduler_cls`, and `lr_scheduler_kwargs`

**Comment by JanFidor** [comment]
Yep, that's exactly what I had in mind, I'll have to thoroughly check out the documentation of kwargs params and see how many another amazing functionalities hide there. 
CL refers to Continual Learning, I mentioned it because I think there's a decent overlap between the problems both it and prediction try to tackle, so incorporating some of the ideas from there in darts might help with tackling more robust datasets. If you'd be interested in the topic, ContinualLearning.ai has an amazing course which I was recommended to get my foot in the door before my collaboration with a research group at my Uni. Personally, I really enjoyed it and I think that the issue can be closed, thanks for the quick reply!
